exercises/object-oriented-programming/src/main/java/com/cbfacademy/accounts/SavingsAccount.java

```diff
@@ -0,0 +1,18 @@
+package com.cbfacademy.accounts;
+
+public class SavingsAccount extends Account{
+
+    public SavingsAccount(double balance, int accountNumber){
+        super(balance, accountNumber);
+
+    }
+
+    @Override
+    public void depositFunds(double interestRate){
+        System.out.println("Your opening balance is £" +getBalance()+".");
+        double interest = ((getBalance() * interestRate) / 100);
+        setBalance(getBalance() + interest);
+        System.out.println("After %"+ interestRate +" interest has been added, your closing balance is £" +getBalance());
+    }   
```

Comment:
This implementation isn't quite what we're expecting. Your savings account effectively can't receive any regular deposits because you've overridden the parent method with this one that only applies interest.

You should add an instance variable and constructor parameter for the interest rate and then create a separate (parameterless) method to allow interest to be added to the balance.